static/app/components/profiling/ProfilingOnboarding/proflingOnboardingSidebar.tsx

```diff
@@ -128,20 +128,30 @@ export function ProfilingOnboardingSidebar(props: CommonSidebarProps) {
     <TaskSidebar orientation={orientation} collapsed={collapsed} hidePanel={hidePanel}>
       <TaskSidebarList>
         <Heading>{t('Profile Code')}</Heading>
-        <CompactSelect
-          triggerLabel={
-            <StyledIdBadge
-              project={currentProject}
-              avatarSize={16}
-              hideOverflow
-              disableLink
-            />
-          }
-          onChange={option => setCurrentProject(option.value)}
-          triggerProps={{'aria-label': currentProject?.slug}}
-          options={projectSelectOptions}
-          position="bottom-end"
-        />
+        <div
+          onClick={e => {
+            // we need to stop bubbling the CompactSelect click event
```

Comment:
😭

Reply:
haha i fixed it in the sidebar but then my tests started failing and i was flipping tables, didn't want to start changing a bunch of unrelated code.